spec/regression/logistics/default-context.json

```diff
@@ -1,3 +1,4 @@
 {
-  "authRoles": ["allusers"]
+  "authRoles": ["allusers"],
+  "childEntityUpdatesViaDictStrategyThreshold": 2
```

Comment:
to me this reads as "the threshold that needs to be reached to activate dict-strategy is 2. So it is active for 2" which is not the case.

Reply:
nevermind, it is the case. I just got confused because the default is supposed to be 3 and this is a file called "default context"

Reply:
yeah I initially had the (real) default at 5 and wanted to reduce it for the tests so we have more chances to trigger it, but I then dialed down the real default to 3. No idea what a good value could be yet.